wandb/data_types.py

```diff
@@ -77,6 +77,17 @@ def history_dict_to_json(run, payload, step=None):
 
     return payload
 
+def cast_numpy_arrays(payload):
```

Comment:
a name like `numpy_arrays_to_lists` or similar would be more consistent with other similar functions in this file.